adapter/internal/oasparser/envoyconf/routes_with_clusters.go

```diff
@@ -514,6 +534,10 @@ func processEndpoints(clusterName string, clusterDetails *model.EndpointCluster,
 		RespectDnsTtl:          conf.Envoy.Upstream.DNS.RespectDNSTtl,
 	}
 
+	if clusterName == rateLimitClusterName {
```

Comment:
please refer to https://github.com/wso2/product-microgateway/pull/3142/files#r1044083874